auth_oidc/models/res_users.py

```diff
@@ -64,6 +65,13 @@ def auth_oauth(self, provider, params):
             _logger.error("No id_token in response.")
             raise AccessDenied()
         validation = oauth_provider._parse_id_token(id_token, access_token)
+        if oauth_provider.data_endpoint:
+            data = requests.get(
+                oauth_provider.data_endpoint,
+                headers={"Authorization": "Bearer %s" % access_token},
+                timeout=10,
+            ).json()
```

Comment:
Error handling for this request is missing?

Reply:
I've pushed a refactoring of this part towards an _auth_oauth_validate function override. It's not possible to reuse it as-is, but at least it becomes really similar